molecule/cookiecutter/scenario/driver/ec2/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -77,8 +77,9 @@
         instance_type: "{{ item.instance_type }}"
         vpc_subnet_id: "{{ item.vpc_subnet_id }}"
         group: "{{ security_group_name }}"
-        instance_tags:
-          instance: "{{ item.name }}"
+        instance_tags: "{{ item.instance_tags | combine({'instance': item.name})
```

Comment:
`instance_tags` needs to be added to the schema, no? Check `molecule/model/schema_v2.py` for more.

Reply:
Not sure exactly what output this gives but it looks fine.

Reply:
There's no schema validation for ec2 driver yet. Should I implement it too?

Reply:
Oh god, really? :face-palm:

Let's just take this PR as is then.